radixdlt-java/src/main/java/com/radixdlt/client/application/translate/unique/UniqueIdAction.java

```diff
@@ -0,0 +1,34 @@
+package com.radixdlt.client.application.translate.unique;
+
+import com.radixdlt.client.application.translate.Action;
+import com.radixdlt.client.atommodel.accounts.RadixAddress;
+import java.util.Objects;
+
+public class UniqueIdAction implements Action {
```

Comment:
`UnqiueIdAction` does not fit in with the other Action names (e.g. `SendMessageAction`). Maybe it should be `AddUniqueIdAction`, or something to make it an **action**.

Reply:
Good Point